go/consensus/tendermint/tendermint.go

```diff
@@ -446,6 +446,24 @@ func (t *tendermintService) StateToGenesis(ctx context.Context, blockHeight int6
 	}, nil
 }
 
+func (t *tendermintService) GetGenesisDocument(ctx context.Context) (*genesisAPI.Document, error) {
```

Comment:
Don't forget to remove the existing `GetGenesis` method that exists in the tendermint-specific service (any callers should use this one).

Reply:
Is this the right thing to name the method?  It's a state dump, not a genesis document (#2886 is illustrative of the fact that there is a difference).

Reply:
After the suggested changes, `GetGenesisDocument` returns `t.genesis`, so it should be equivalent to the old `GetGenesis` method.